doc/classes/DisplayServer.xml

```diff
@@ -719,6 +719,12 @@
 				[b]Note:[/b] This method is implemented on macOS, Windows and Linux (X11).
 			</description>
 		</method>
+		<method name="is_touchscreen_available" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if touch events are available (Android or iOS) or if [member ProjectSettings.input_devices/pointing/emulate_touch_from_mouse] is [code]true[/code].
```

Comment:
```suggestion
				Returns [code]true[/code] if touch events are available (Android or iOS).
```

`emulate_touch_from_mouse` is checked inside input, and won't have any effect on `is_touchscreen_available`, so this is wrong.

Reply:
Please have a look at `DisplayServer::is_touchscreen_available` where `Input::get_singleton()->is_emulating_touch_from_mouse()` is checked.
This returned `true`, if it was set during initialization in
https://github.com/godotengine/godot/blob/c241f1c52386b21cf2df936ee927740a06970db6/main/main.cpp#L2234-L2246
which is governed by `ProjectSettings.input_devices/pointing/emulate_touch_from_mouse`.

This behavior and documentation was there before this PR and I didn't change change it.

Reply:
Yes, but it's a virtual method that is overridden, so it will only work if it's not implemented (e.g., won't do anything on Web where it has a real implementation).

Reply:
So I guess this check should be copied to the web override at least for docs to be correct.

Reply:
I have updated the Web-implementation and the docs to reflect the implementation. I have checked, that it is overridden only on android, iOS and web.
Thanks for your review!